android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -858,12 +873,14 @@ else if (e.type == ExoPlaybackException.TYPE_SOURCE) {
     }
 
     private static boolean isBehindLiveWindow(ExoPlaybackException e) {
+        com.google.android.exoplayer2.util.Log.e("ExoPlayer Exception", e.toString());
```

Comment:
any reason you're using Log from exoplayer?

Reply:
No particular reason actually. Should I change it to `Log.e`?

Reply:
Changed to `Log.e()` 👍